<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MapTab</class>
 <widget class="QWidget" name="MapTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>541</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,20">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tilePage">
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="TilePickerWidget" name="tilePickerWidget" native="true"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="eventsPage">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLineEdit" name="lineEdit"/>
           </item>
           <item>
            <widget class="QTableView" name="tableView"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="toggleMapTableButton">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../../assets/icons.qrc">
           <normaloff>:/icons8-expand-48.png</normaloff>:/icons8-expand-48.png</iconset>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mapInfoWidget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="mapNameFilter"/>
          </item>
          <item>
           <widget class="BaseTable" name="mapInfoTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="alternatingRowColors">
             <bool>true</bool>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <property name="sortingEnabled">
             <bool>true</bool>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>32</number>
            </attribute>
            <attribute name="horizontalHeaderHighlightSections">
             <bool>false</bool>
            </attribute>
            <attribute name="horizontalHeaderStretchLastSection">
             <bool>true</bool>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QWidget" name="mapToolBar" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="layerCheckBox_0">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">visibleLayersButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="layerCheckBox_1">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">visibleLayersButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="layerCheckBox_2">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">visibleLayersButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="layerCheckBox_3">
            <property name="text">
             <string/>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">visibleLayersButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="layerCheckBox_4">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">visibleLayersButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="layerCheckBox_5">
            <property name="text">
             <string/>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">visibleLayersButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="tileModeButton">
            <property name="text">
             <string>Tiles</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">modeButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="eventModeButton">
            <property name="text">
             <string>Events</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">modeButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pickerModeButton">
            <property name="text">
             <string>Picker</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">modeButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_5">
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="layerButton_1">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">paintLayerButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="layerButton_2">
            <property name="maximumSize">
             <size>
              <width>32</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>2</string>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">paintLayerButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>85</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="runButton">
            <property name="text">
             <string>Run</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="MapView" name="mapView">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="backgroundBrush">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>61</red>
            <green>56</green>
            <blue>70</blue>
           </color>
          </brush>
         </property>
         <property name="interactive">
          <bool>false</bool>
         </property>
         <property name="cacheMode">
          <set>QGraphicsView::CacheBackground</set>
         </property>
         <property name="viewportUpdateMode">
          <enum>QGraphicsView::SmartViewportUpdate</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BaseTable</class>
   <extends>QTableView</extends>
   <header>base_table.hpp</header>
  </customwidget>
  <customwidget>
   <class>MapView</class>
   <extends>QGraphicsView</extends>
   <header>map_view.hpp</header>
  </customwidget>
  <customwidget>
   <class>TilePickerWidget</class>
   <extends>QWidget</extends>
   <header>tile_picker_widget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../assets/icons.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="paintLayerButtonGroup"/>
  <buttongroup name="modeButtonGroup"/>
  <buttongroup name="visibleLayersButtonGroup">
   <property name="exclusive">
    <bool>false</bool>
   </property>
  </buttongroup>
 </buttongroups>
</ui>
